graviti/manager/branch.py

```diff
@@ -102,6 +104,10 @@ def create(self, name: str, revision: Optional[str] = None) -> Branch:
             name=name,
             revision=revision,
         )
+
+        if getattr(head, "name", None) == revision and head.commit_id != response["commit_id"]:
+            warnings.warn(StatusWarning(type(head).__name__.lower(), revision), stacklevel=2)
+
         return Branch(self._dataset, **response)
 
     def get(self, name: str) -> Branch:
```

Comment:
do we need a default value for the `name` in `BranchManager.get`

Reply:
1. Get the default branch
2. If the HEAD is branch, get the current branch. Otherwise, the `name` is required. In my opinion, this is better.